selfdrive/controls/controlsd.py

```diff
@@ -206,6 +206,9 @@ def update_events(self, CS):
                                                  LaneChangeState.laneChangeFinishing]:
       self.events.add(EventName.laneChange)
 
+    if self.sm['lateralPlan'].laneChangeState == LaneChangeState.belowSpeed:
+      self.events.add(EventName.belowSteerSpeed)
```

Comment:
isn't belowsteerspeed for vehicles that can't steer below a certain speed? See "No ALC Below" in the readme.

Reply:
Oh yes , thanks :) In that case I need to create a new event.